src/platform/utilities/oauth/constants.js

```diff
@@ -29,6 +29,7 @@ export const CLIENT_IDS = {
   VAWEB: 'vaweb',
   VAMOBILE: 'vamobile',
   VAMOCK: 'vamock',
+  ARP: 'arp',
```

Comment:
@asg5704 I believe that an ARP-specific client ID is required for the backend to identify the correct ClientConfig that results in ARP-specific behavior (the logout redirect URI for instance). I think soon it will also power new behavior around distinguishing that a request is authenticated for ARP but not for VA and vice versa. The codepaths I looked at in `vets-website` seem like they may default to VA web's client ID but that that would be incorrect for these reasons.

Also I just noticed that (if I am understanding the need for an ARP client ID correctly) [this codepath](https://github.com/department-of-veterans-affairs/vets-website/blob/3c0ccd317dcfdbe548b9a3bd3c210b07c5741159/src/platform/utilities/oauth/utilities.js#L178) for requesting the access token only supports the VA web client ID, so I think something will have to change around that. Maybe `AuthApp` is capable of working for other clients if its use of sessionStorage expands to include another data point beyond just the `authReturnUrl`? Or maybe leveraging `AuthApp` is untenable (but I suspect it isn't)?

Reply:
You are mostly correct as I just confirmed with the BE on the `client_id` portion. I confirmed with the BE that the `client_id` will be some random-hash that will need to be stored AND that the `/token` request does not require a `client_id` (I'll get this removed)

So here is what I'm recommending
- `dev` => keep the `ARP: 'arp'`
- `staging` => update the `staging.config.js` as follows
```jsx
oAuthOptions: {
  ...arpWebOAuthOptions,
  clientId: `<hash-from-client-config>`
}
```

Reply:
I have that value now for staging, if I give it here will he be able to incorporate it into this PR? Or does there need to be a refactor of this setting first?

Reply:
From my team there is no great rush on getting everything corrected outside of localhost.

Reply:
Yes, he will be able to incorporate it into this PR by following the recommendations below

> So here is what I'm recommending
> 
> * `dev` => keep the `ARP: 'arp'`
> * `staging` => update the `staging.config.js` as follows
> 
> ```js
> oAuthOptions: {
>   ...arpWebOAuthOptions,
>   clientId: `<hash-from-client-config>`
> }
> ```

Reply:
here's the staging client_id: `ce6db4d7974daf061dccdd21ba9add14`